content/ru/accessability.html

```diff
@@ -0,0 +1,77 @@
+
+<p><strong>Content</strong></p>
+<ul>
+  <li><a href="#statement">Информация о обеспечении доступности</a></li>
+  <li><a href="#how-we-support-and-maintain">Как мы поддерживаем и обеспечиваем доступность </a></li>
+  <li><a href="#our-standards">Наши стандарты</a></li>
+  <li><a href="#help-feedback-and-formal-complaints">Помощь, отзывы и жалобы</a></li>
+</ul>
+
+
+<h2 id="statement">Заявление о обеспечении доступности
+</h2>
+<p>Команда vote.gov стремится создать доступный веб-сайт для всех избирателей, включая людей, которые испытывают трудности со зрением, слухом, работой с компьютерным оборудованием, имеют когнитивные проблемы или трудности в обучении. Мы также стремимся охватить недостаточно защищенных людей, в том числе тех, чей основной язык не английский.  Обеспечение доступности - это постоянная работа, и мы стремимся улучшить наш сайт, проводя регулярные тренинги для сотрудников, которые пишут наш контент, создают наши документы и разрабатывают наш веб-сайт.p>
+
+<h2 id="how-we-support-and-maintain">Как мы поддерживаем и обеспечиваем доступность
+</h2>
+<p><strong>Доступность веб-сайта vote.gov включает: </strong></p>
+<ul>
+  <li>Ручное тестирование содержимого на возможность чтения с клавиатуры и программ чтения с экрана</li>
+  <li>Использование семантических заглавий разделов для организации контента
+  </li>
+  <li>Предоставление людям возможности использовать клавиатуру для доступа ко всем ссылкам и интерактивным частям нашего веб-сайта
+  </li>
+  <li>Работу функции «перейти к основной странице»
+  </li>
+  <li>Добавление кода к внешним ссылкам для объявления «открытие в новом окне» для программ чтения с экрана
+  </li>
+  <li>Предоставление подробного альтернативного текста для изображений, значков и логотипов
+  </li>
+  <li>Привлечение реальных людей к переводу содержания веб-сайта для обеспечения контекста и улучшения опыта взаимодействия
+  </li>
+  <li>Предоставление пользователям возможности изменять размер текста согласно их предпочтениям
+  </li>
+</ul>
+
+<h2 id="our-standards">Наши стандарты доступности</h2>
+<p>Мы разрабатываем наши веб-страницы в соответствии со стандартами <a href="https://www.section508.gov/">Раздела 508﻿ </a> или более высокими стандартами, которые являются техническими требованиями, гарантирующими соблюдение нами федерального закона Раздел 508.  Мы также соблюдаем Руководство по обеспечению доступности веб-сайта W3C - WCAG 2.1, которые можно найти на <a href="https://www.w3.org/tr/wcag21">W3C website</a>. Мы соответствуем стандартам уровня AA, что означает, что наш контент доступен большинству людей в большинстве случаев.</p>
```

Comment:
for links that go to untranslated pages we need to add in the shortcode ` {{< inenglish >}}`